src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Tabs/combinations.html.twig

```diff
@@ -41,11 +41,14 @@
   }|json_encode }}"
   ></div>
 
-  {{ render(controller('PrestaShopBundle:Admin\\Sell\\Catalog\\Product\\Combination:paginatedList', {
-    'productId': productId,
-  })) }}
+    {{ render(controller('PrestaShopBundle:Admin\\Sell\\Catalog\\Product\\Combination:paginatedList', {
+      'productId': productId,
+    })) }}
 
-  <div id="combinations-stock-availability">
-    {#    @todo: render stock availability preferences here when extracted to a reusable component. check StockType.php#}
+    <div id="combinations-stock-availability">
+      {#    @todo: render stock availability preferences here when extracted to a reusable component. check StockType.php#}
```

Comment:
Todo always up to date ?

Reply:
May be link it to an issue ?

Reply:
I didn't have time yet, but this will be replaced by an Unavailable block